apps/dashboard/src/routes/invite/s/course/[hash]/+page.svelte

```diff
@@ -146,7 +144,7 @@
       label="Join Course"
       type="submit"
       isDisabled={disableSubmit || loading}
-      isLoading={loading || !$profile.id}
+      isLoading={loading}
```

Comment:
why did you remove this?

Reply:
was referencing what we have on bellcurves codebase, this was deleted over there but i've resolved it in this PR